src/inflation_utils.py

```diff
@@ -0,0 +1,47 @@
+# Inflation Data is currently only from Canada
+
+import numpy as np 
+import json
+from requests import post
+from datetime import datetime
+
+class InflationUtils:
+	def __init__(self, start_date, end_date = str(datetime.date(datetime.now()))):
```

Comment:
The end_date = str(datetime.date(datetime.now())) would normally be =None then check for None in the init, this looks a bit confusing

Reply:
Meaning something like this?
```
def __init__(self,start_date,end_date):
    if end_date is None:
        end_date = str(datetime.date(datetime.now()))
```

Reply:
```
def __init__(self,start_date,end_date=None):
    if end_date is None:
        end_date = str(datetime.date(datetime.now()))
```

Reply:
That's what I would do since end_date shouldn't have to be specified

Reply:
Right, makes sense!

Reply:
I've already merged this so no need to make the changes, you could create another PR if you like but its all good :)

Reply:
Truu, just working on the actual inflation adjustment stuff now so I figured I'd fix these at the same time